applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -78,6 +78,39 @@ def test_Naca0012SmallCompressible(self):
             for file_name in os.listdir():
                 if file_name.endswith(".time"):
                     kratos_utilities.DeleteFileIfExisting(file_name)
```

Comment:
this [exists in the core](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/kratos_utilities.py#L27-L32)